### PR TITLE
 - Add `parseValue` config option to InputObjectType to parse input value to custom value object

### PR DESCRIPTION
**Pull request by ruudk**
This was removed in 9791dc1e1be817f44e41dfdb01abc98b108f6f92 but for some projects
it is useful to keep the schema sorted.

### CODE REVIEW

File: src/Utils/SchemaPrinter.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
 /**
  * Prints the contents of a Schema in schema definition language.
  *
- * @phpstan-type Options array{}
+ * @phpstan-type Options array{sortTypes?: bool}
  * @psalm-type Options array<never, never>
```
Comment (spawnia) [inline src/Utils/SchemaPrinter.php:38]:
```suggestion
```

File: tests/Utils/SchemaPrinterTest.php [hunk 2]
```diff
@@ -21,11 +21,14 @@
  */
 final class SchemaPrinterTest extends TestCase
 {
-    private static function assertPrintedSchemaEquals(string $expected, Schema $schema): void
+    /**
+     * @param array<string, mixed>  $options
```
Comment (spawnia) [inline tests/Utils/SchemaPrinterTest.php:25]: Should use type import to annotate this as `Options`

**Comment by ruudk** [comment]
Ready :)

**Comment by spawnia** [comment]
Oops, messed up the commit message. Thanks for the PR!

**Comment by ruudk** [comment]
What happened here? A merge went wrong because the commit doesn't reflect what it did :)

**Comment by ruudk** [comment]
Might be good to quickly fix with a force push 🙏

**Comment by spawnia** [comment]
Nah, I do not force push on `master`.

**Comment by spawnia** [comment]
I squashed the PR while merging, but added the wrong commit message - I copied the wrong line from resolving conflicts in the changelog.

**Pull request merged**